tasks/utilities/task_broad_terra_tools.wdl

```diff
@@ -175,9 +179,9 @@ task export_taxon_tables {
clean2}\t~{num_reads_clean_pairs}\t~{trimmomatic_version}\t~{bbduk_docker}\t~{r1_mean_q}\t~{r2_mean_q}\t~{assembly_fasta}\t~{contigs_gfa}\t~{shovill_pe_version}\t~{shovill_se_version}\t~{quast_report}\t~{quast_version}\t~{genome_length}\t~{number_contigs}\t~{n50_value}\t~{cg_pipeline_report}\t~{cg_pipeline_docker}\t~{est_coverage}\t~{gambit_report}\t~{gambit_predicted_taxon}\t~{gambit_predicted_taxon_rank}\t~{gambit_closest_genomes}\t~{gambit_version}\t~{gambit_db_version}\t~{gambit_docker}\t~{busco_version}\t~{busco_database}\t~{busco_results}\t~{busco_report}\t~{ts_mlst_results}\t~{ts_mlst_predicted_st}\t~{ts_mlst_pubmlst_scheme}\t~{ts_mlst_version}\t~{serotypefinder_report}\t~{serotypefinder_docker}\t~{serotypefinder_serotype}\t~{ectyper_results}\t~{ectyper_version}\t~{ectyper_predicted_serotype}\t~{lissero_results}\t~{lissero_version}\t~{lissero_serotype}\t~{sistr_results}\t~{sistr_allele_json}\t~{sister_allele_fasta}\t~{sistr_cgmlst}\t~{sistr_version}\t~{sistr_predicted_serotype}\t~{seqsero2_report}\t~{seqsero2_version}\t~{seqsero2_predicted_antigenic_profile}\t~{seqsero2_predicted_serotype}\t~{seqsero2_predicted_contamination}\t~{kleborate_output_file}\t~{kleborate_version}\t~{kleborate_key_resistance_genes}\t~{kleborate_genomic_resistance_mutations}\t~{kleborate_mlst_sequence_type}\t~{kleborate_ktype}\t~{kleborate_otype}\t~{kleborate_klocus_confidence}\t ~{kleborate_olocus_confidence}\t~{legsta_results}\t~{legsta_predicted_sbt}\t~{legsta_version}\t~{tbprofiler_output_file}\t~{tbprofiler_output_bam}\t~{tbprofiler_output_bai}\t~{tbprofiler_version}\t~{tbprofiler_main_lineage}\t~{tbprofiler_sub_lineage}\t~{tbprofiler_dr_type}\t~{tbprofiler_resistance_genes}\t~{amrfinderplus_all_report}\t~{amrfinderplus_amr_report}\t~{amrfinderplus_stress_report}\t~{amrfinderplus_virulence_report}\t~{amrfinderplus_version}\t~{amrfinderplus_db_version}\t~{amrfinderplus_amr_genes}\t~{amrfinderplus_stress_genes}\t~{amrfinderplus_virulence_genes}\t~{genotyphi_report_tsv}\t~{genotyphi_mykrobe_json}\t~{genotyphi_version}\t~{genotyphi_species}\t~{genotyphi_st_probes_percent_coverage}\t~{genotyphi_final_genotype}\t~{genotyphi_genotype_confidence}\t~{ani_highest_percent}\t~{ani_highest_percent_bases_aligned}\t~{ani_output_tsv}\t~{ani_top_species_match}\t~{ani_mummer_version}\t~{resfinder_pheno_table}\t~{resfinder_pheno_table_species}\t~{resfinder_seqs}\t~{resfinder_results}\t~{resfinder_pointfinder_pheno_table}\t~{resfinder_pointfinder_results}\t~{resfinder_db_version}\t~{resfinder_docker}\t~{prokka_gff}\t~{prokka_gbk}\t~{prokka_sqn}\t~{plasmidfinder_plasmids}\t~{plasmidfinder_results}\t~{plasmidfinder_seqs}\t~{plasmidfinder_docker}\t~{plasmidfinder_db_version}\t~{pbptyper_predicted_1A_2B_2X}\t~{pbptyper_pbptype_predicted_tsv}\t~{pbptyper_version}\t~{poppunk_gps_cluster}\t~{poppunk_gps_external_cluster_csv}\t~{poppunk_version}\t~{seroba_version}\t~{seroba_serotype}\t~{seroba_ariba_serotype}\t~{seroba_ariba_identity}\t~{seroba_details}"  >> ~{samplename}_terra_table.tsv
```

Comment:
Just a formatting comment, there is a space before the "~" in {kleborate_klocus_confidence}\t ~

Reply:
good catch, I will fix this. Luckily this didn't break things

Reply:
I found a few more of these 🤭